components/PriceFeedTable.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState, useCallback } from "react";
+import { StyledTd } from "./Table";
+
+interface PriceFeed {
+  ids: string;
+  assetType: string;
+  name: string;
+}
+
+interface TableColumnWidths {
+  assetType: string;
+  name: string;
+  ids: any;
+}
+
+const columnWidths: TableColumnWidths = {
+  assetType: "w-3/10",
+  name: "w-1/5",
+  ids: "w-1/2",
+};
+
+const PriceFeedTable = ({ priceFeeds }: { priceFeeds: PriceFeed[] }) => {
+  const [selectedAssetType, setSelectedAssetType] = useState<string>("All");
+  const [copiedId, setCopiedId] = useState<string | null>(null);
```

Comment:
I generally recommend using `undefined` instead of `null`, see https://github.com/sindresorhus/meta/discussions/7 for a good discussion on why